### ISSUE TITLE
GNUSTEP_USER_ROOT is obsolete should be a warning....

### ISSUE THREAD

**Issue by gcasa** [gnustep/tools-make#22]
```
/usr/GNUstep/System/Library/Makefiles/config-noarch.make:141: *** GNUSTEP_USER_ROOT is obsolete.  Stop.
This is gnustep-make 2.9.0. Type 'make print-gnustep-make-help' for help.
Running in gnustep-make version 2 strict mode.
cd .; \
/usr/GNUstep/System/Library/Makefiles/mkinstalldirs ./obj
```

I am not certain why this is an error.   There are some older scripts which set this, but it ultimately may not be used.  What is the harm in making it a warning?

**Comment by gcasa** [comment]
This variable is actually being set via windowmaker.  So when you start an app with wmaker's dock, it inherits the environment.   They have corrected this in the latest version 0.95.9.

**Comment by gcasa** [comment]
This is not a bug in gnustep-make.
